internal/cli/sketch/new.go

```diff
@@ -38,7 +39,14 @@ func initNewCommand() *cobra.Command {
 		Long:    tr("Create a new Sketch"),
 		Example: "  " + os.Args[0] + " sketch new MultiBlinker",
 		Args:    cobra.ExactArgs(1),
-		Run:     func(cmd *cobra.Command, args []string) { runNewCommand(args, overwrite) },
+		Run:     func(cmd *cobra.Command, args []string) {
+			re := regexp.MustCompile("^[a-zA-Z].")
+			if !re.MatchString(args[0]) {
+				fmt.Println("Error: Value can only contain alphabetic characters")
```

Comment:
```suggestion
				fmt.Println("Error: Value can only contain alphabetic characters")
```
The error message is incorrect. You can see an appropriate error message here:

https://github.com/arduino/arduino-ide/blob/2cb9d64f307c6b4f8a0d0bd652c6cac2058f090f/arduino-ide-extension/src/common/protocol/sketches-service.ts#L152